server/express-router.md

```diff
@@ -0,0 +1,96 @@
+Previously, we've learnt to separate routes into their own file, by exporting them as functions and requiring them within our `server.js` file:
```

Comment:
I would suspect we might not separate the `app.get` routes into their own separate file on the day we introduce Express. I suspect we'll just put everything in `server.js`.